app/controllers/accounts/connected_accounts/selected_email_controller.rb

```diff
@@ -14,6 +14,7 @@ def edit
         @select_email_form = build_select_email_form
         @can_add_email = EmailPolicy.new(current_user).can_add_email?
         analytics.sp_select_email_visited
+        @email_id = @identity.email_address_id || last_email
```

Comment:
What is this change for?

Reply:
I see. I think I was able to implement that.

Reply:
> It substitutes the user's last signed-in email if there isn't an existing selected email address so the checked value is set. This is consistent experience with the select email from in the provider consent flow.

I don't think I'd necessarily expect that behavior, or at least I don't know that it's strictly within scope of the bug fix. It might be worth further discussion with the team if you'd like to propose that change.

My take on it is that if I sign in to my account with a specific email address, that doesn't necessarily connote a preference to use that email with a partner when managing connections through my account dashboard in the same way as through the consent screen. When I'm selecting through the consent screen, that's after I've made a decision to use a specific email when signing in to access that particular partner.

Reply:
That makes sense to me. I have removed that default.